src/main/java/com/creativemd/littletiles/LittleTiles.java

```diff
@@ -64,7 +64,7 @@ public class LittleTiles {
     public static final String modid = "littletiles";
     public static final String version = LTTags.VERSION;
 
-    public static int maxNewTiles = 51200;
+    public static int maxNewTiles = 512;
```

Comment:
Why this change? It seems unrelated to the rest of the PR

Reply:
> just make a new commit on the branch to revert that change and it'll be ok

Not sure what you mean. It's supposed to be 512, but my last PR accidentally changed it. Should I leave the fix in this PR or make a new one?